tools/libSwiftScan/SwiftCaching.cpp

```diff
@@ -270,11 +286,12 @@ computeCacheKeyFromIndex(llvm::cas::ObjectStore &CAS,
 swiftscan_string_ref_t
 swiftscan_cache_compute_key(swiftscan_cas_t cas, int argc, const char **argv,
                             const char *input, swiftscan_string_ref_t *error) {
-  std::vector<const char *> Compilation;
-  for (int i = 0; i < argc; ++i)
-    Compilation.push_back(argv[i]);
+  llvm::SmallVector<const char *> Compilation;
```

Comment:
Nit: I would rename this "ArgsStorage" or similar to avoid accidentally using this.  Same below.